scripts/clear_pycache

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo find . \( -name __pycache__ -o -name '*.pyc' \) | xargs rm -rf
```

Comment:
I wonder if we can be a little smarter and use `sudo` only when it's actually needed. But it's not a blocker for this PR, just a random thought.